rocket_twin/systems/station/station.py

```diff
@@ -0,0 +1,24 @@
+from cosapp.base import System
+
+from rocket_twin.systems import Pipe, Rocket, Tank
+
+
+class Station(System):
+    """A space station composed by a rocket, a tank and a pipe connecting them.
+
+    Inputs
+    ------
+
+    Outputs
+    ------
+    """
+
+    def setup(self):
+        self.add_child(Pipe("pipe"))
+        self.add_child(Tank("g_tank"))
```

Comment:
it should be first tank than pipe to limite problem size

Reply:
Check